pkg/check/info.go

```diff
@@ -0,0 +1,37 @@
+package check
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	versionLabel = "version"
+)
+
+var (
+	vCenterInfoMetric = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "vsphere_vcenter_info",
+			Help:           "Information about vSphere vCenter.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{versionLabel},
+	)
+)
+
+func init() {
+	legacyregistry.MustRegister(vCenterInfoMetric)
```

Comment:
Can we just use plain Prometheus client libs?  I am not sure we want to carry the entire k8s burden of metric in this operator.

Reply:
Kubernetes metrics are already pulled by library-go, which exposes workqueue lengths and other stuff. I actually started with `prometheus.MustRegister`, but I did not see the metrics.